tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -1262,6 +1263,40 @@ public void EnsurePublicApiSurfaceExistsOnInterface()
             CheckBuilderCommonMethods(byRefreshTokenBuilder);
         }
 
+        [TestMethod]
+        public void CheckUserProvidedCorrelationIDTest()
+        {
+            using (var harness = CreateTestHarness())
+            {
+                harness.HttpManager.AddInstanceDiscoveryMockHandler();
+                var correlationId = new Guid();

```

Comment:
Guid.NewGuid()

Reply:
new Guid() always returns Guid.Null (00000-00000-00000-...)

Reply:
Missed this one. thanks